app/src/workspace/workspaceActions.js

```diff
@@ -113,7 +113,10 @@ export function saveWorkspace(errorAction) {
       if (newLayer.subtype === LAYER_TYPES.Encounters) {
         newLayer.type = LAYER_TYPES.Encounters;
       }
+      // TODO Should we use a whitelist of fields instead ?
```

Comment:
@nerik convert this question into a ticket. Don't mind leaving the `TODO`, but if there is not ticket is hard to remember about this unless you step into this line of code again. And when you step into it, you are for sure working in another thing so you can't spend time working on it.

Reply:
@enriquetuya Ok. Since it's purely front-end, would you create it in GFW-Tasks?